index.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" media="screen" href="css/style.css" />
+    <title>Hello, world!</title>
+</head>
+
+<body>
+    <main id="main-doc">
+        <div class="row">
+            <div class="col-md-2">
+                <nav id="navbar" class="navbar navbar-light bg-light">
+                    <header>Hamilton Bay Steam Iron</header>
+                    <a class="nav-link" href="#Parts_and_features">Parts and features</a>
+                    <a class="nav-link" href="#How_to_dry_iron">How to dry iron</a>
+                    <a class="nav-link" href="#How_to_steam_iron">How to steam iron</a>
+                    <a class="nav-link" href="#Care_and_cleaning">Care and cleaning</a>
+                    <a class="nav-link" href="#Self_Clean">Self Clean</a>
+
+                </nav>
+            </div>
+            <div class="col-md-10">
+                <section class="main-section border border-5" id="Parts_and_features">
+                    <header>
+                        <h1>Parts and features</h1>
+                    </header>
+                    <p>The iron consists of the following parts:</p>
+                    <img src="resources/plancha.JPG">
+                    <code style="background-color: #f1f1f1;">
+                        <ol>
+                          <li>Blast/Spray Buttons </li>   
+                          <li>Adjustable Steam Switch</li> 
+                          <li>Water Tank Opening</li> 
+                          <li>Spray Nozzle</li> 
+                          <li>Soleplate</li> 
+                          <li>Water Window</li> 
+                          <li>Power On Light</li> 
+                          <li>Heel Rest</li> 
+                          <li>Cord Retraction Button</li>
+                          <li>Temperature Control Dial</li> 
+                        </ol> 
+        
+                    </code>
+                    <p>Notice the retractable cord is located on the device's behind.</p>
+                </section>
+                <section class="main-section  border border-5" id="How_to_dry_iron">
+                    <header>
+                        <h1>How to dry iron</h1>
```

Comment:
- [ ] `<h1>` tags are meant to be used only once as a title for the website. Switch them for  `<h2>`  tags instead

Reply:
It's Ok here